examples/client/client.c

```diff
@@ -2502,7 +2525,13 @@ THREAD_RETURN WOLFSSL_THREAD client_test(void* args)
 
     #ifdef HAVE_ECC
         /* load ecc verify too, echoserver uses it by default w/ ecc */
-        #ifndef TEST_LOAD_BUFFER
+        #ifdef NO_FILESYSTEM
+        if (wolfSSL_CTX_load_verify_buffer(ctx, ca_ecc_cert_der_256,
```

Comment:
On Visual Studio this is unhappy `ca_ecc_cert_der_256`. I think you need to force define `USE_CERT_BUFFERS_256` above at line 50 as well.

```
Error | C2065 | 'ca_ecc_cert_der_256': undeclared identifier | testsuite | z:\davidgarske\github\wolfssl\examples\client\client.c | 2529 | 
```